config/locales/en.yml

```
@@ -184,6 +184,8 @@ en:
     apply: Apply
     clear: Clear
     close_menu: Close menu
+    country:
+      title: Country
```

```suggestion
      title: Location
```

I know it's less accurate, but businesses have been asking for "location search" so I think this will read better to them.